### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: bootloader/src/pyi_pythonlib.c [hunk 1]
```diff
@@ -454,7 +454,7 @@ pyi_pylib_start_python(ARCHIVE_STATUS *status)
      * That is, the APIs use the string pointer as given and will neither copy
      * its contents nor free its memory.
      *
-     * NOTE: Statics are zero-initialized. */
+     * NOTE: Statistics are zero-initialized. */
```
Comment (bwoodsend) [inline bootloader/src/pyi_pythonlib.c]: The original is correct here (although it would be better written as `Static variables are zero-initialized`).

**Review by bwoodsend — CHANGES_REQUESTED** [review]
Nice. You'll have to rewrap the lines that [the linter is complaining](https://github.com/pyinstaller/pyinstaller/runs/6212494998?check_suite_focus=true) about which are now slightly longer than our 120 character limit.

**Comment by bwoodsend** [comment]
Thanks @kianmeng

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**